### DISCUSSION TITLE
特殊弹幕屏蔽依然失效

### DISCUSSION THREAD

**Discussion by ChenJW431** [the1812/Bilibili-Evolved#4876] [综合讨论 / General]
![屏幕截图 2024-08-16 103711](https://github.com/user-attachments/assets/c2ec9d7d-fd13-4b50-b1a0-562d1512bae2)
![屏幕截图 2024-08-16 103722](https://github.com/user-attachments/assets/c91384df-c793-44a6-9b43-b78680671ed7)
火狐129，脚本版本2.9.1

**Comment by the1812** [comment]
这是弹窗 另一个功能可以屏蔽

**Reply by ChenJW431** [reply]
![屏幕截图 2024-08-16 105110](https://github.com/user-attachments/assets/318b920c-4b1e-48c1-a271-0d825cbe8d62)
请问是这个吗，还是不好用

**Reply by the1812** [reply]
需要更新这个组件，你的版本还是5月的

**Reply by ChenJW431** [reply]
> 需要更新这个组件，你的版本还是5月的

好了，谢谢
